script/misc/update-wiki-pages

```diff
@@ -8,6 +8,7 @@ use Pod::Usage;
 use MediaWiki::API;
 use Test::More qw(no_plan);
 use YAML::XS qw(Dump);
+# NOTE: on macos, LWP::Protocol::https also needs to be installed
```

Comment:
There's nothing macos specific about that - the https protocol handler is in a separate CPAN module and is in a separate package for most if not all linux distributions as well.

But because the protocol handlers are loaded dynamically they don't need to be explicitly required here and won't be a dependency of LWP and hence of anything that requires LWP.

Reply:
thanks for the explaination, I've removed this comment